### PR TITLE
Feature/r-genefilter updated to version 1.72.0

### PR DESCRIPTION
**Pull request by alexander-baumgarten**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-genefilter/package.py [hunk 1]
```diff
@@ -3,16 +3,43 @@
...
 
 
 class RGenefilter(RPackage):
+
     """genefilter: methods for filtering genes from high-throughput
        experiments."""
 
     homepage = "https://bioconductor.org/packages/genefilter"
-    git      = "https://git.bioconductor.org/packages/genefilter.git"
+    #  git      = "https://git.bioconductor.org/packages/genefilter.git"
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/r-genefilter/package.py]: What's the reason behind changing from the git commits to the release archive ?

File: var/spack/repos/builtin/packages/r-genefilter/package.py [hunk 2]
```diff
@@ -3,16 +3,43 @@
...
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/r-genefilter/package.py]:
```suggestion
```

File: var/spack/repos/builtin/packages/r-genefilter/package.py (hunk 1 above)
Comment (alexander-baumgarten) [reply var/spack/repos/builtin/packages/r-genefilter/package.py]: Ran into some problems with the git commits, working on changing it back now.

File: var/spack/repos/builtin/packages/r-genefilter/package.py [hunk 3]
```diff
@@ -7,12 +7,15 @@
 
 
 class RGenefilter(RPackage):
+
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/r-genefilter/package.py]:
```suggestion
```

File: var/spack/repos/builtin/packages/r-genefilter/package.py [hunk 4]
```diff
@@ -7,12 +7,15 @@
...
     """genefilter: methods for filtering genes from high-throughput
        experiments."""
 
     homepage = "https://bioconductor.org/packages/genefilter"
     git      = "https://git.bioconductor.org/packages/genefilter.git"
+    # url      = "https://bioconductor.org/packages/release/bioc/src/contrib/genefilter_1.72.0.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/r-genefilter/package.py]: Why adding a commented out url?

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by adamjstewart** [comment]
Hmm, https://bioconductor.org/packages/release/bioc/html/genefilter.html only lists the previous release. Anyway, it looks like the dependencies are out-of-date and need to be updated.

File: var/spack/repos/builtin/packages/r-genefilter/package.py [hunk 5]
```diff
@@ -13,6 +13,7 @@ class RGenefilter(RPackage):
     homepage = "https://bioconductor.org/packages/genefilter"
     git      = "https://git.bioconductor.org/packages/genefilter.git"
 
+    version('1.73.0', commit='154bc907d2d84884ab3c5c19c1da73d3ba61aad7')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-genefilter/package.py:16]:
```suggestion
    version('1.72.0', commit='154bc907d2d84884ab3c5c19c1da73d3ba61aad7')
```
?

**Pull request closed without merging**